Furball.Vixie/Graphics/Backends/OpenGL41/QuadRendererGL41.cs

```diff
@@ -75,7 +75,7 @@ private struct InstanceData {
             this._boundTextures = new TextureGL41[this._backend.QueryMaxTextureUnits()];
 
             string vertSource = ResourceHelpers.GetStringResource("ShaderCode/OpenGL41/InstancedRenderer/VertexShader.glsl");
-            string fragSource = ResourceHelpers.GetStringResource("ShaderCode/OpenGL41/InstancedRenderer/FragmentShader.glsl");
+            string fragSource = ShadersGL41.GetFragment(backend);
```

Comment:
I really dislike this, like what shader is this getting? before i looked at what this does I didnt even know why it was there, this needs to be way more specific

Reply:
do this everywhere else aswell